ballista/rust/core/src/serde/logical_plan/to_proto.rs

```diff
@@ -973,24 +973,24 @@ impl TryInto<protobuf::LogicalPlanNode> for &LogicalPlan {
                     )),
                 })
             }
-            LogicalPlan::Analyze { verbose, input, .. } => {
-                let input: protobuf::LogicalPlanNode = input.as_ref().try_into()?;
+            LogicalPlan::Analyze(a) => {
```

Comment:
It's better to import the Analyze struct  and use this code style 
```
LogicalPlan::Analyze(Aanlyze{
 verbose, input,....
})
```

Reply:
In fact, I don't have a strong desire which is better. When the struct has many variables, I'd like to use `StructVar(s) => {s.v1, s.v2 ...}`

Reply:
I agree either style seems fine to me